client/state/data-layer/wpcom/email-forwarding/add/index.js

```diff
@@ -62,7 +64,7 @@ export const addEmailForwardFailure = ( action, error ) => {
 };
 
 export const addEmailForwardSuccess = ( action, response ) => {
-	const { domainName, mailbox, destination } = action;
+	const { domainName, mailbox, destination, onSuccessRedirect = noop } = action;
```

Comment:
It doesn't look like we're supplying this property any more. If so, I think we should remove it and the related handling.